src/main/java/com/kodilla/ecommercee/domain/Cart.java

```diff
@@ -21,10 +21,10 @@
     @Column(name = "CART_ID")
     private long id;
 
-    @OneToOne
+    @OneToOne(cascade = CascadeType.ALL)
```

Comment:
Te same uwagi co w pozostałych pullRequestach, wygląda na to, że kilka z nich zawiera te same zmiany